app/backend/src/couchers/models.py

```diff
@@ -656,18 +656,21 @@ def is_valid(self):
         return (self.created <= func.now()) & (self.expiry >= func.now())
 
 
+communities_seq = Sequence("communities_seq")
+
+
 class Node(Base):
     """
     Node, i.e. geographical subdivision of the world
     """
 
     __tablename__ = "nodes"
 
-    id = Column(BigInteger, primary_key=True)
+    id = Column(BigInteger, communities_seq, primary_key=True)
```

Comment:
What does a Sequence do?

Reply:
It's a "source of autoincrementing numbers", so this all makes the community objects basically have globally unique IDs across the different tables. There's some discussion about this though.